embed_grader/serapis/views/courses.py

```diff
@@ -169,6 +171,88 @@ def unenroll_course(request, course_id):
     }
     return render(request, 'serapis/unenroll_course.html', template_context)
 
+@login_required(login_url='/login/')
+def download_csv(request, course_id):
+    user = User.objects.get(username=request.user)
+    course = Course.objects.get(id=course_id)
+    print(course)
+    students = []
+    headers = ['Last Name', 'First Name', 'UID']
+
+    response = HttpResponse(content_type='text/csv')
+    response['Content-Disposition'] = 'attachment; filename="%s_scores.csv"' % course
+    writer = csv.writer(response)
+
+    if not user.has_perm('download_csv', course):
+        return HttpResponse("Not enough privilege.")
+
+    assignment_list = Assignment.objects.filter(course_fk=course_id).order_by('-id')
+    for h in assignment_list:
+        headers.append(h.name)
+
+    writer.writerow(headers)
+
+    assignment_ids = []
```

Comment:
`assignment_ids` seems not necessary since you can always get the IDs by going through `assignment list`